tests/bluetooth/tester/src/gatt.c

```diff
@@ -181,7 +181,7 @@ static uint8_t btp2bt_uuid(const uint8_t *uuid, uint8_t len,
 
 static void supported_commands(uint8_t *data, uint16_t len)
 {
-	uint8_t cmds[4];
+	uint8_t cmds[5];
```

Comment:
I also noticed now that this length was too short since the new opcode is `0x20`.